docs/TYPES.md

```diff
@@ -12,34 +12,127 @@ Each triggered `track` can and must be of a certain type, to be properly display
   - [`traffic`](#traffic)
```

Comment:
I would prefer the toc to match the order of contents:

  - [`music`](#music)
  - [`live`](#live)
  - [`news`](#news)
  - [`weather`](#weather)
  - [`traffic`](#traffic)
  - [`audio`](#audio)
  - [`commercial`](#commercial)
  - [`jingle`](#jingle)

Reply:
Sure, but I'd like to get feedback on the new order (ordered by importance /semantically close types together)